cirq_superstaq/daily_integration_test.py

```diff
@@ -51,8 +51,25 @@ def test_aqt_compile(service: cirq_superstaq.Service) -> None:
         cirq.rx(np.pi / 2)(qubits[4]),
         cirq.rz(np.pi / 2)(qubits[4]),
     )
-    assert service.aqt_compile(circuit).circuit == expected
-    assert service.aqt_compile([circuit]).circuits == [expected]
+
+    cirq.testing.assert_circuits_with_terminal_measurements_are_equivalent(
+        service.aqt_compile(circuit).circuit, expected, atol=1e-08
```

Comment:
do we need `expected` anymore or could this just be
```suggestion
        service.aqt_compile(circuit).circuit, circuit, atol=1e-08
```

Reply:
How so? `expected` is different from `circuit`

`circuit = cirq.Circuit(cirq.H(qubits[4]))` vs 
 ```
expected = cirq.Circuit(
        cirq.rz(np.pi / 2)(qubits[4]),
        cirq.rx(np.pi / 2)(qubits[4]),
        cirq.rz(np.pi / 2)(qubits[4]),
    )```

Reply:
i believe `assert_circuits_with_terminal_measurements_are_equivalent` only compares unitaries, which should generally be equivalent

maybe this isn't what we want in this case though, if e.g. we're trying to test that we get the "optimized" hadamard decomposition (i.e. `rz`, `rx`, `rz`) instead of the less canonical (`rx`, `rz`, `rx`)

Reply:
Ah, not familiar with the guts assert_circuits_with_terminal_measurements_are_equivalent`. Should we revert this given that we are indeed trying to assert a certain optimization?

Reply:
ah maybe in that case? in most cases it seems pretty nice because it should be robust against future updates/optimizations (e.g. i can imagine there being subtle changes in the compiled version of the the more complex `test_cq_compile` circuit below) but in this one case maybe the circuit is simple enough that we shouldn't expect anything to change

or maybe another option is to keep `assert_circuits_with_terminal_measurements_are_equivalent`, and instead of comparing to a specific circuit add something that just checks that there's only one `rx` gate in the compiled circuit

Reply:
I'm fine with leaving as-is. Internal checks should make sure that things are being optimized appropriately.

Reply:
i agree. in that case i think `expected` definitely isn't necessary here & below

Reply:
Done.